mod.conf

```diff
@@ -1,4 +1,4 @@
 name = torch_bomb
 description = Place torches throughout your entire surroundings with a torch bomb
-depends =
-optional_depends = mcl_core, mcl_sounds, mcl_explosions, mcl_tnt, default, tnt, creative, farming
\ No newline at end of file
+optional_depends = mcl_core, mcl_sounds, mcl_explosions, mcl_tnt, default, tnt, creative, farming
+min_minetest_version = 5.3.0
```

Comment:
Why 5.3.0?

Reply:
as mentioned in the PR description:
> Bump minimal minetest version to 5.3 (the mod uses now minetest.is_creative_enabled())

mt 5.3.0 is the first version w/ this function.

Reply:
I see. Thank you for pointing this out, I seem to have skipped right to the diff tab.